README.md

```diff
@@ -54,70 +59,73 @@ Before any calls are made to the api though you need to get authenticate and rec
 which you will use for all other communication.
 
 #### Endpoints
-|Endpoint|Description|
-|---|---|
-|api-token-auth/|Authentication Endpoint, get your tokens here|
-|risk-types/|Create, Read, Update and Delete RiskTypes here.|
-|risks/|Create, Read, Update and Delete Risks here.|
 
-         
+| Endpoint        | Description                                     |
+| --------------- | ----------------------------------------------- |
+| api-token-auth/ | Authentication Endpoint, get your tokens here   |
```

Comment:
where you're saying 'here', should there be a link?

Reply:
No, you get your token at this current  endpoint, I'll give it a second look and reword since it's unclear.